src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -27,3 +27,11 @@ Filecoin Storage Mining Subsystem
 # Sector in StorageMiner State Machine (both)
 
 {{< diagram src="diagrams/sector_fsm.dot.svg" title="Sector State Machine (both)" >}}
+
+# ElectionPoSt
+
+To enable short PoSt response time, miners are required submit a PoSt when they are elected to mine a block, hence PoSt on election or ElectionPoSt. When miners win a block, they need to immediately generate a PoStProof and submit that along with the ElectionProof. Both the ElectionProof and PoStProof are checked at Block Validation by `StoragePowerConsenusSubsystem`. When a block is included, a special message is added that calls `SubmitElectionPoSt` which will process sector updates in the same way as successful `SubmitSurprisePoSt` do. Note that it will be a single PoStProof in a single `SubmitElectionPoSt` message instead of a Commit-then-Prove scheme.
```

Comment:
This should discuss:
- the construction of the post itself (what goes into it, how we sample challenges, how we prove it, etc -- maybe wait till that finalizes)
- how we create the message to put into the VM

Reply:
will update readme when more of ElectionPoSt finalizes